lib/simulator-xcode-6.js

```diff
@@ -227,8 +229,37 @@ class SimulatorXcode6 {
     await killAllSimulators();
   }
 
+  async endSimulatorDaemon () {
+    // Looks for launchd daemons corresponding to the sim udid and tries to stop them cleanly
+    // This prevents xcrun simctl erase hangs.
+    log.debug(`Killing any simulator daemons for ${this.udid}`);
+    try {
+      let stopCmd = `launchctl list | grep ${this.udid} | cut -f 3 | xargs -n 1 launchctl stop`;
+      await exec('bash', ['-c', stopCmd]);
+    } catch (err) {
+      log.warn('Could not stop simulator daemons, carrying on anyway!');
+    }
+    try {
+      let removeCmd = `launchctl list | grep ${this.udid} | cut -f 3 | xargs -n 1 launchctl remove`;
```

Comment:
fair amount of duplication in these magic strings; should we not just to `'${magic} stop'` and `'${magic} remove'`?